components/notice/README.md

```diff
@@ -0,0 +1,10 @@
+This component is used to implement notices in editor.
```

Comment:
Anything in "components" is not meant to be specific to the editor.

Currently, adding notices externally is not very intuitive, but I think we might look to leveraging the [data module](https://github.com/WordPress/gutenberg/tree/master/data) for notices to become an independent unit that can be manipulated externally (and reused on any future screens).

Reply:
Is there anything actionable I should do at this point, to improve the README for example?

Reply:
It's fine. The README can be updated when notices are refactored.